.github/workflows/gradle.yml

```diff
@@ -13,7 +13,6 @@ jobs:
     strategy:
       matrix:
         java:
-          - 11
```

Comment:
Is java 11 causing the issue here ?

Reply:
Not sure, but unlikely. I see some of the build tasks by @kiranprakash154  having succeeded:
https://github.com/opensearch-project/performance-analyzer/actions/workflows/gradle.yml
The gradle tasks weren't skipped, so they do appear to have run normally.

Reply:
I don't think so. Bwc tests run with jdk11 and jdk17 every time and both fail. All the successful runs I saw in that list are on 2.x or 1.x branches which use old bwcVersion = 1.1 (changed to 2.1 now)